public/app/plugins/datasource/prometheus/completer.ts

```diff
@@ -276,9 +276,11 @@ export class PromCompleter {
       }
       query = '{__name__' + op + '"' + expr + '"}';
     }
-    return this.datasource.performInstantQuery({ expr: query }, new Date().getTime() / 1000).then(response => {
-      this.labelQueryCache[expr] = response.data.data.result;
-      return response.data.data.result;
+    let range = this.datasource.getTimeRange();
```

Comment:
Should be `const { start, end } = this.datasource.getTimeRange();`
Also, `url` below should be `const`.